### PR TITLE
Fix typo: Update maven update command and description

### PR DESCRIPTION
**Pull request by JohnNiang**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

typo fix.

- How to fix?

Replace with maven wrapper command and fix compile typo.

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: docs/en/guides/How-to-build.md [hunk 1]
```diff
@@ -133,6 +133,6 @@ Refer to [Build docker image](../../../docker) for more details.
 </dependency>
 ```
 5. Run `./mvnw compile -Dmaven.test.skip=true`
-6. Run `maven update`. Must remove the clean projects item before maven update(This will be clear the proto conversion Java file generated by the complie)
+6. Run `./mvnw update`. Must remove the clean projects item before maven update(This will be clear the proto conversion Java file generated by the compile)
```
Comment (wu-sheng) [inline docs/en/guides/How-to-build.md]: `./mvnw update` is not a command.

Comment (JaredTan95) [reply docs/en/guides/How-to-build.md]: may be `./mvnw clean` ?

Comment (wu-sheng) [reply docs/en/guides/How-to-build.md]: To be honest, I am not sure what does this part mean. As I never use Eclipse these days.

Comment (JohnNiang) [reply docs/en/guides/How-to-build.md]: > `./mvnw update` is not a command.

Got it. It's a feature only on Eclipse.

Comment (JaredTan95) [reply docs/en/guides/How-to-build.md]: If so, We should keep it without chenges. :P

**Comment by wu-sheng** [comment]
You don't need to close, the `compile` is really a typo. Just don't change this command.

**Comment by wu-sheng** [comment]
I revert that part of the change for you. `compile` should be `compiling`.

**Comment by JohnNiang** [comment]
Thank you for rectification.

**Comment by JaredTan95** [comment]
199th contributor~

**Comment by wu-sheng** [comment]
> 199th contributor~

Yes, we are closing the 200th. Who will own the honor? :)

**Comment by wu-sheng** [comment]
Strange, a markdown change makes the CI unstable...

**Comment by wu-sheng** [comment]
Is there anything wrong about downloading JDBC driver?

**Comment by JaredTan95** [comment]
I re-run the ci.

**Comment by wu-sheng** [comment]
> I re-run the ci.

All errors are about no suitable driver of MySQL, but it should be downloaded in CI process.

**Comment by JaredTan95** [comment]
> [ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.30.0:start (start) on project e2e-mysql: I/O Error: [skyapm/e2e-container:jdk8-1.3] "skywalking-e2e-container-local-single-node-mysql": Timeout after 300399 ms while waiting on url http://localhost:32771/e2e/health-check -> [Help 1]

It looks like oap server  container stared failed...

**Comment by JohnNiang** [comment]
> > [ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.30.0:start (start) on project e2e-mysql: I/O Error: [skyapm/e2e-container:jdk8-1.3] "skywalking-e2e-container-local-single-node-mysql": Timeout after 300399 ms while waiting on url http://localhost:32771/e2e/health-check -> [Help 1]
> 
> It looks like oap server container stared failed...

Maybe we shall check the `bin/oapService.sh` bash file because of <https://github.com/apache/skywalking/pull/4241/checks?check_run_id=392573262#step:6:280>.

**Comment by JaredTan95** [comment]
@JohnNiang  you delete your fork? I noted `unknown repository`

**Comment by JohnNiang** [comment]
> @JohnNiang you delete your fork? I noted `unknown repository`

I will re-request this PR soon.

**Pull request closed without merging**